IDEAS/Fluid/Taps/Examples/TapProfile.mo

```diff
@@ -0,0 +1,49 @@
+within IDEAS.Fluid.Taps.Examples;
+model TapProfile "Example with two 'TapProfile' models"
+  extends Modelica.Icons.Example;
+  package Medium = IDEAS.Media.Water;
+
+  IDEAS.Fluid.Taps.TapProfile tapProfile1(redeclare package Medium = Medium)
+    annotation (Placement(transformation(extent={{-10,10},{10,30}})));
+  IDEAS.Fluid.Taps.TapProfile tapProfile2(redeclare package Medium = Medium,
```

Comment:
It's a detail, but maybe vertically flip this model?

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/7b8ca7927a8a9bee1b5f4db61ff5aa4005a69d2a